### PR TITLE
More checks

### PR DESCRIPTION
**Pull request by doubleface**
This PR depends on #57 

- It compares the current travis config file to the one from the connector template (fetched from github).
- Everything must be identical in travis config file, the repo and secure keys excepted
- it also compares version in the manifest and the package.json
- it checks the declaration of the connector in the registry
- it checks the assets directory and that the icon specified in the manifest exist in this directory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,8 @@
     "js-yaml": "3.11.0",
     "lodash": "4.17.5",
     "mkdirp": "0.5.1",
+    "request": "^2.85.0",
```
Comment (aenario) [inline package.json]: strict versioning ?

File: src/check.js [hunk 2]
```diff
@@ -18,6 +19,38 @@ const lintedByEslintPrettier = {
...
+      'categories',
+      'fields',
+      'data_types',
+      'permissions',
+      'developer',
+      'langs',
+      'locales'
+    ]
+
+    const missingFields = mandatoryFields.filter(field => !info.manifest[field])
```
Comment (sebn) [inline src/check.js]: Maybe `info.manifest[field] == null` or `info.manifest[field] === undefined`? (assuming some future field with `false` value could make sense)

File: src/check.js [hunk 3]
```diff
@@ -29,39 +64,71 @@ const hasFieldsInManifest = {
...
     )
+
+    const blackList = ['repo']
+    travis.deploy.forEach((deploy, index) => {
+      assert(
+        deepEquals(
+          Object.keys(deploy),
+          Object.keys(templateTravis.deploy[index])
+        ),
+        `${index}: deploy keys should the same as the connector template`
```
Comment (sebn) [inline src/check.js]: Typo

File: src/check.js [hunk 4]
```diff
@@ -84,19 +151,54 @@ const renovateIsConfigured = {
...
+    return true
   },
-  nickname: '4branches',
-  message: 'Repository should have 4 branches.',
-  link: 'https://github.com/konnectors/docs/blob/master/status.md#git-branch'
+  nickname: 'assets',
+  message: 'The ./assets directory should exist and be configured in webpack'
+}
+
+const manifestAndPackageJsonSameVersion = {
```
Comment (sebn) [inline src/check.js]: Would it make sense to just remove the version from the `package.json` file? Or are we supposed to install connectors through npm at some point?

Comment (doubleface) [reply src/check.js]: No, the registry also checks the version in `package.json` file (I tried :-))

**Pull request merged**